blazor/rich-text-editor/tools/dialogSettings.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: DialogSettings in Blazor RichTextEditor Component | Syncfusion
+description: Checkout and learn here all about DialogSettings in Syncfusion Blazor RichTextEditor component and more.
+platform: Blazor
+control: RichTextEditor
+documentation: ug
+---
+
+# DialogSettings in Blazor RichTextEditor Component
```

Comment:
Customization of Blazor Rich Text Editor Dialog

Reply:
addressed